composer.json

```diff
@@ -14,9 +14,17 @@
     "friendsofphp/php-cs-fixer": "^2.5",
     "jakub-onderka/php-console-highlighter": "^0.3.2",
     "jakub-onderka/php-parallel-lint": "^0.9.2",
+    "localheinz/composer-normalize": "^0.1.0",
```

Comment:
this should go in require-dev right ?

Reply:
Normally yes, but this package is a meta package to be added to `require-dev` it self.